src/main/java/com/radixdlt/identifiers/RadixAddress.java

```diff
@@ -46,17 +47,23 @@
 	 */
 	private final Supplier<String> base58 = Suppliers.memoize(this::computeBase58);
 
-	public RadixAddress(byte magic, ECPublicKey key) {
-		this.key = Objects.requireNonNull(key);
+	/**
+	 * The magic byte of this address
+	 */
+	private final transient int magicByte; // MIGRATION: From Client Library
```

Comment:
Same comment about this comment as above, don't think it's relevant.

Reply:
Any comments with "MIGRATION" is to be removed by me :D I was gonna do it "last minute", just there to help me keep track of things, but yes, you are right, I should not forget to remove them :)